custombackend/src/main/java/com/algolia/custombackend/MainActivity.java

```diff
@@ -0,0 +1,31 @@
+package com.algolia.custombackend;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import com.algolia.custombackend.elasticbackend.ElasticTransformer;
+import com.algolia.instantsearch.helpers.InstantSearch;
+import com.algolia.instantsearch.helpers.Searcher;
+import com.algolia.instantsearch.ui.views.SearchBox;
+import com.algolia.search.saas.Index;
+
+public class MainActivity extends AppCompatActivity {
+
+    private static final String ALGOLIA_APP_ID = "latency";
```

Comment:
I don't think we need these credentials if you don't target algolia

Reply:
Removed